tox.ini

```diff
@@ -1,8 +1,16 @@
 [tox]
-envlist = pypy, pypy3, py27, py34, py35, py36
+envlist = pypy, pypy3, py27, py34, py35, py36, flake8
 
 [testenv]
 deps = -rrequirements-dev.txt
 commands =
     py.test sasstests.py
+
+[testenv:flake8]
+deps= flake8
+skip_install=True
+commands=
     flake8 .
```

Comment:
Hmm, `unicode` is a good one. Since the other way around (`flake8` only on `py3x`) also isn't a good solution, one possibility would be doing `{py27,py34}-flake8`. This does complicate things slightly.

I normally deal with py3-only projects, which doesn't come with all this hassle 😞 .